src/kendo.numerictextbox.js

```diff
@@ -501,12 +513,27 @@ var __meta__ = { // jshint ignore:line
 
                 e.preventDefault();
             } else if ((min !== null && min >= 0 && value.charAt(0) === "-") || !isValid) {
+                that._blinkInvalidState();
                 e.preventDefault();
             }
 
             that._key = 0;
         },
 
+        _blinkInvalidState: function () {
+            var that = this,
+                wrapper = that._inputWrapper.addClass(INVALID);
+
+            that._validationIcon.show();
+            clearTimeout(that._blinkTimeout);
+
+            that._blinkTimeout = setTimeout(function(){

```

Comment:
Can you re-check why this is happening? Maybe you need to ensure that the `this` object is the correct one. There should be no difference between `setTimeout` and `window.setTimeout` as calling `setTimeout` will search for such a method in the scope chain and actually end in `window.setTimeout` as there is no "local" method, which overrides it.

In addition, run the tests and make sure that the focus stays in the browser that runs the tests, as changing the focused element might be the problem.